### PR TITLE
Expose scen

### PR DESCRIPTION
**Pull request by MirijaHa**
Exposing baseline and shock scenario as argument to the user. Currently there is no choice between different scenarios but as soon as the data is ready we can activate it. Snapshot tests pass.

FYI @jakubcervenka please have a look as well, as it affects IO. I can wait with emrging till your vignette PR is merged so that you do not have to solve additional merge conflicts.

### CODE REVIEW

File: vignettes/articles/03-run-stress-test.Rmd [hunk 1]
```diff
@@ -88,6 +88,9 @@ stress_test_arguments
 ### Understanding the input arguments
 The meaning of each of the detail arguments is as follows:
 
+* `baseline_scenario` holding the name of the baseline scenario.
```
Comment (jacobvjk) [inline vignettes/articles/03-run-stress-test.Rmd:91]: Maybe we can extend this to explain that we use a business-as-usual scenario as the baseline scenario?

`baseline_scenario` holds the name of the baseline scenario to be used in the stress test. We use a business-as-usual scenario for the baseline.

File: vignettes/articles/03-run-stress-test.Rmd [hunk 2]
```diff
@@ -88,6 +88,9 @@ stress_test_arguments
...
+* `shock_scenario` holding the name of the scenario that companies are shocked
```
Comment (jacobvjk) [inline vignettes/articles/03-run-stress-test.Rmd:92]: similar as above:
`shock_scenario` holds the name of the scenario that companies are forced to transition to via a late and sudden shock. We use an ambitious transition scenario as the shock scenario.

**Review by jacobvjk — CHANGES_REQUESTED** [review]
see comments on vignette, else works like a charm

**Comment by jacobvjk** [comment]
also pls merge master again

File: vignettes/articles/03-run-stress-test.Rmd (hunk 1 above)
Comment (jakubcervenka) [reply vignettes/articles/03-run-stress-test.Rmd:91]: agree

Comment (MirijaHa) [reply vignettes/articles/03-run-stress-test.Rmd]: done

File: vignettes/articles/03-run-stress-test.Rmd (hunk 2 above)
Comment (MirijaHa) [reply vignettes/articles/03-run-stress-test.Rmd]: done

File: vignettes/articles/03-run-stress-test.Rmd [hunk 3]
```diff
@@ -88,6 +88,11 @@ stress_test_arguments
 ### Understanding the input arguments
 The meaning of each of the detail arguments is as follows:
 
+* `baseline_scenario` holds the name of the baseline scenario to be used in the
+stress test. We use a business-as-usual scenario for the baseline.
+* `shock_scenario`holds the name of the scenario that companies are forced to
```
Comment (jacobvjk) [inline vignettes/articles/03-run-stress-test.Rmd:93]: missed a space after the variable name. Not trying to be overly picky, up to you to fix before merging

**Review by jacobvjk — APPROVED** [review]
LGTM
tiny nit

**Pull request merged**